runtime-testsuite/test/org/antlr/v4/test/runtime/java/api/perf/TimeLexerSpeed.java

```diff
@@ -84,18 +77,13 @@ public static void main(String[] args) throws Exception {
 		tests.compilerWarmUp(100);
 
 		int n = 3500;
-		URL sampleJavaFile = TimeLexerSpeed.class.getClassLoader().getResource(Parser_java_file);
-		URL sampleFile = TimeLexerSpeed.class.getClassLoader().getResource(PerfDir+"/udhr_hin.txt");
-		tests.load_legacy_java_ascii(n);
```

Comment:
this legacy load should probably still be here. I'm updating in a branch here...